base_partner_sequence/models/partner.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2004-2009 Tiny SPRL (<https://tiny.be>).
+# Copyright 2013 initOS GmbH & Co. KG (<https://www.initos.com>).
+# Copyright 2016 Tecnativa - Vicent Cubells
+# Copyright 2016 Camptocamp - Akim Juillerat (<https://www.camptocamp.com>).
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, exceptions, models
+
+
+class ResPartner(models.Model):
+    """Assigns 'ref' from a sequence on creation and copying"""
+
+    _inherit = "res.partner"
+
+    def _get_next_ref(self, vals=None):
```

Comment:
Why do we need `vals=None` in here? Because I don't see that you are using `vals` in function
Is there any module that depends to this module and need `vals`? In that case, Please add comment for this.
By the way, should we need `@api.model` for this function?

Reply:
@quannguyen1560  code updated base on your comments.